provisioner/roles/configure_qradar_instance/tasks/post_instantiation_config.yml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
This seems wrong. This code is needed to register the Check Point NGFW with the Check Point MGMT server. That has nothing to do at all with QRadar?!